addons/mass_mailing/views/mailing_contact_views.xml

```diff
@@ -116,14 +114,14 @@
                                 <field name="email" widget="email"/>
                                 <field name="is_blacklisted" invisible="1"/>
                             </div>
-                            <field name="company_name"/>
-                            <field name="country_id" options="{'no_open': True, 'no_create': True}"/>
+                            <field name="tag_ids" widget="many2many_tags" options="{'color_field': 'color', 'on_tag_click': 'edit_color'}"
+                                   placeholder="e.g. Roadshow" style="width: 100%"/>
                         </group>
                         <group>
-                            <field name="create_date" readonly="1" invisible="not id"/>
-                            <field name="message_bounce" readonly="1" invisible="not id"/>
-                            <field name="tag_ids" widget="many2many_tags" options="{'color_field': 'color', 'on_tag_click': 'edit_color'}"
-                                   placeholder="e.g. &quot;VIP&quot;, &quot;Roadshow&quot;, ..." style="width: 100%"/>
+                            <field name="company_name" placeholder="e.g. Azure Interior"/>
+                            <field name="country_id" options="{'no_open': True, 'no_create': True}"/>
+                            <field name="create_date" string="Create Date" readonly="1" invisible="not id"/>
+                            <field name="message_bounce" readonly="1" invisible="not id or message_bounce == 0"/>
```

Comment:
Can we have message_bounce be more than 0 for non-created record? Can probably simplify.